utility/CalculateInstalledMemory.hpp

```diff
@@ -0,0 +1,38 @@
+/**
+ *   Copyright 2015 Pivotal Software, Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ **/
+
+#ifndef QUICKSTEP_CALCULATE_INSTALLED_MEMORY_HPP_
```

Comment:
The header guard should be ``QUICKSTEP_UTILITY_CALCULATE_INSTALLED_MEMORY_HPP_``